cluster/defaults.go

```diff
@@ -268,6 +272,28 @@ func (c *Cluster) setClusterNetworkDefaults() {
 	}
 }
 
+func (c *Cluster) setClusterAuthnDefaults() {
+	setDefaultIfEmpty(&c.Authentication.Strategy, DefaultAuthStrategy)
+
+	for _, S := range strings.Split(c.Authentication.Strategy, "|") {
```

Comment:
@erikwilson why did you have to use`|` again? Would really prefer `,` separated list.

Reply:
Using capital `S` vs `s` is a bad pattern. Can you use more meaningful names?

Reply:
`|` was used because default values in golang tags can not contain `,` - webhook is not currently set as a default strategy so we can change it back to `,` if desired, but may cause issues for a default delimitated value in the future.

Will fix variable naming.